python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -352,14 +365,33 @@ async def latiss_acquire(self):
         # Setup instrument and telescope
         # The following code sets up the instrument and slews the telescope
         # simultaneously.
-        tmp, data = await asyncio.gather(
-            self.atcs.slew_object(
+        if self.object_name:
```

Comment:
I think you want should check that ra and dec are not None, if so, then use object name, e.g.;

```
if self.object_ra is not None and self.object_dec is not None:
    # call slew_icrs, but pass in the name of the target as `target_name` so it 
    # goes to the header.

    ....
else:
    # Slew object
```

Reply:
re-arranged this section a bit. Essentially tries to use coords and falls back to object name.
Else statement removed since object_name is now mandatory.